eng/Packages.Data.props

```diff
@@ -203,6 +203,14 @@
     <PackageReference Update="Microsoft.AspNetCore.Http.Features" Version="[2.1.1,6.0)" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.AI.Agents'))">
```

Comment:
Is the intent to allow these to evolve independently?   Seems that we could bundle the floating version into a single group with a compound condition.

Reply:
This block is now removed and we combined into a single group via a wildcard.